docs/buildah-bud.md

```diff
@@ -591,6 +592,19 @@ will convert /foo into a `shared` mount point.  The propagation properties of th
 mount can be changed directly. For instance if `/` is the source mount for
 `/foo`, then use `mount --make-shared /` to convert `/` into a `shared` mount.
 
+If you use the special `temp=` option, the source directory from the host
+will be mounted as a temporary volume using a COW (Copy On Write) mount into
+the container.  Modifications to this mountpoint will allowed and stored on the
```

Comment:
s/will allowed/will be allowed/  or remove "will allowed and" and replace it with "be"